internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -197,6 +197,13 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"public_ip_type": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				Default:      string(kusto.PublicIPTypeIPv4),
+				ValidateFunc: validation.StringInSlice([]string{string(kusto.PublicIPTypeIPv4), string(kusto.PublicIPTypeDualStack)}, false),
```

Comment:
Maybe we can organize it like the following to make it more readable.
```
ValidateFunc: validation.StringInSlice([]string{
     string(kusto.PublicIPTypeIPv4), 
     string(kusto.PublicIPTypeDualStack),
   }, false),
```

Reply:
Done